content/en/apps/guides/forms/wealth-quintiles.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Tracking Wealth Quintiles"
+linkTitle: "Quintiles"
+weight: 
+description: >
+  How to track wealth quintiles on the profile of each family member in the household
+relatedContent: >
+
+---
+
+
+A few things are needed to get the data copied onto the family members. 
+
+1. A report submitted with either `doc.fields.NationalQuintile` or `doc.fields.UrbanQuintile` or both. 
+1. An offline user that has the permission of `can_write_wealth_quintiles`. 
+1. Once the form is submitted the contact will then get the fields `"wealth_quintile_national"` and ` "wealth_quintile_urban"` if they are provided in the form.
+
+
+### Sample Report
+
+
+``` json
+{
+  "form": "family_survey",
+  "type": "data_record",
+  "content_type": "xml",
+  "fields": {
+    "place_id": "23c7ab6e-6ea5-4f9f-9ffa-f00a1e30278d",
+    "place_name": "Medic Area",
+    "NationalQuintile": "5",
+    "UrbanQuintile": "3"
```

Comment:
Ideally would these be strings, numbers, or ints?

Reply:
I'm not 100% sure as I just used a form that was already built. I'm going to generate a sample form as requested above and try the different data types.

Reply:
@garethbowen  setting the type on the xlsform doesn't set the actual type. It always seems to be a string. I had prompted a question about storing the specific type to the report but did not get an answer. Do we have a way to store the data in a specific type?

Reply:
Not sure... Don't let me block this PR. If I find a way I can update later...